src/app/tests/suites/commands/system/SystemCommands.cpp

```diff
@@ -53,7 +53,7 @@ CHIP_ERROR SystemCommands::Reboot(uint16_t discriminator)
     constexpr const char * scriptName = "Reboot.py";
 
     char command[128];
```

Comment:
If you end up adding more options we need to make sure the size if fine. We should probably use a `constexpr` at the beginning of the file and makes it a bit bigger than `128`.